src/guessTheNumber.js

```diff
@@ -0,0 +1,18 @@
+const min = Math.ceil(1)
+const max = Math.floor(100)
+const randomNum = Math.floor(Math.random() * (max - min + 1) + min)
```

Comment:
This could be simplified. Currently it is doing a silly calculation: `Math.random() * (100 - 1 + 1) + 1)`